tools/generator/src/Generator/AddBazelDependenciesTarget.swift

```diff
@@ -396,6 +408,8 @@ elif [ "${ENABLE_PREVIEWS:-}" == "YES" ]; then
     --swiftcopt=-Xfrontend
     --swiftcopt=-enable-dynamic-replacement-chaining
   )
+else
+  config=rules_xcodeproj_build
```

Comment:
For users, should these 4 different configs be kept in sync as much as possible in order to minimize thrashing the bazel analysis cache? If so, would it make sense for all these to inherit from a base config like `rules_xcodeproj_base`?

Reply:
They all inherit from `rules_xcodeproj`.

Reply:
https://github.com/buildbuddy-io/rules_xcodeproj/blob/c842b5970392f2749c7690a7f169d63508fde097/xcodeproj/internal/bazel_integration_files/xcodeproj.bazelrc#L1-L7

Reply:
ok great. I hadn't gotten to reading that yet. Maybe this should also be in the documentation when you get to that part :)